client/src/AuthService.js

```diff
@@ -0,0 +1,65 @@
+export default class AuthService {
```

Comment:
I think it's better not to use a class here and export the functions individually, since most of the time you'll only be needing fetch, but not the other methods